pkg/commands/blast.go

```diff
@@ -36,10 +36,23 @@ func Blast(c *cli.Context) error {
 
 	api := slack.New(slackToken)
 
-	slackChannels, err := api.GetChannels(true)
+	slackChannels := []slack.Channel{}
+	types := []string{"public_channel", "private_channel"}
+	err := error(nil)
+	cursor := ""
 
-	if err != nil {
-		return err
+	// Loop through pagination till we get all slack channels.
+	for hasMorePages := true; hasMorePages; hasMorePages = !(cursor == "") {
+		slackChannelsPage := []slack.Channel{}
+		params := slack.GetConversationsParameters{cursor, "true", 1000, types}
+
+		slackChannelsPage, cursor, err = api.GetConversations(&params)
+
+		if err != nil {
+			return err
+		}
+
+		slackChannels = append(slackChannels, slackChannelsPage...)
```

Comment:
Pull out the code copied from channels and convert into a function that's used in both places.